libs/features/georoutingv2/georoutingv2.css

```diff
@@ -57,6 +63,7 @@
 
 .dialog-modal.locale-modal-v2 .link-wrapper {
   line-height: 48px;
+  position: relative;
```

Comment:
do we need this?

Reply:
Yes, the picker has to open in relative to the `link-wrapper` otherwise it opens relative to its parent container `georouting-wrapper`

Reply:
I'm not quite getting you - the picker's position is absolute which means it's positioned relative to its nearest positioned ancestor, which is the link-wrapper.
Can you show me an example where this `position: relative` is necessary?